docker/build/x86_64/Dockerfile-CUDA

```diff
@@ -0,0 +1,53 @@
+# This dockerfile should be built with `make docker-build` from the stash root.
+
+# Build Frontend
+FROM node:alpine as frontend
+RUN apk add --no-cache make
+## cache node_modules separately
+COPY ./ui/v2.5/package.json ./ui/v2.5/yarn.lock /stash/ui/v2.5/
+WORKDIR /stash
+RUN yarn --cwd ui/v2.5 install --frozen-lockfile.
+COPY Makefile /stash/
+COPY ./graphql /stash/graphql/
+COPY ./ui /stash/ui/
+RUN make generate-frontend
+ARG GITHASH
+ARG STASH_VERSION
+RUN BUILD_DATE=$(date +"%Y-%m-%d %H:%M:%S") make ui
+
+# Build Backend
+FROM golang:1.19-bullseye as backend
+RUN apt update && apt install -y build-essential golang
+WORKDIR /stash
+COPY ./go* ./*.go Makefile gqlgen.yml .gqlgenc.yml /stash/
+COPY ./scripts /stash/scripts/
+COPY ./vendor /stash/vendor/
+COPY ./pkg /stash/pkg/
+COPY ./cmd /stash/cmd
+COPY ./internal /stash/internal
+COPY --from=frontend /stash /stash/
+RUN make generate-backend
+ARG GITHASH
+ARG STASH_VERSION
+RUN make build
```

Comment:
This is duplicated from the `Dockerfile` in the same directory. Could we not make this the build docker image use `FROM stash/build` then add the stuff below?

Reply:
The main problem is that only the frontend can be shared.
The backend must be built on a glibc platform such that it can run on Ubuntu.
Where alpine uses musl and makes Ubuntu incompatible binaries.

Reply:
Sorry didn't realise it needed a different platform. Can you revert https://github.com/stashapp/stash/pull/3419/commits/444badbd49a31311503e6a807cb5522c8d142ebe?

Reply:
> Sorry didn't realise it needed a different platform. Can you revert [444badb](https://github.com/stashapp/stash/commit/444badbd49a31311503e6a807cb5522c8d142ebe)?

Reverted it